content/_layouts/frame.html.haml

```diff
@@ -55,7 +55,7 @@
         :media => 'screen'}/
 
     / Non-obtrusive CSS styles
-    %link{:href => "#{site.base_url}/assets/bower/ionicons/ionicons.css",
+    %link{:href => "#{site.base_url}/assets/bower/ionicons/css/ionicons.min.css",
```

Comment:
How it was would have never worked. this meant all the icons on the landing page were missing.